hooks/pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+echo "Running lint..."
+golangci-lint run
+
+echo "Running tests..."
+go test ./...
```

Comment:
Lets not enable tests by default yet. Tests are still largely meaningless since our fork of geth. Once we've refactored the tests, we can turn this back on and require passing tests for PRs

Reply:
added a prompt with defaults that should resolve this